views/templates/admin/options_certificate.tpl

```diff
@@ -40,17 +40,17 @@
 	    
 	    <div class="form-group">
                 <label class="control-label col-lg-3">{l s='Language' mod='trustedshops'}</label>
-                <div class="col-lg-4">{$lang|escape:'html':'UTF-8'}</div>
+                <div class="col-lg-4"><output style="font-size: 13px;">{$lang|escape:'html':'UTF-8'}</output></div>
```

Comment:
As you can see, you wrote `style="font-size: 13px;"` for all your output tags. You should move these style definition in a CSS file.